app/models/admin/scholarship_group.rb

```diff
@@ -1,5 +1,7 @@
 class Admin::ScholarshipGroup < ApplicationRecord
   validates_presence_of :name
   validates_uniqueness_of :name
-  has_many :admin_scholarships, :class_name => 'Admin::Scholarship',inverse_of: :admin_scholarship_group
+  #has_many :admin_scholarships, :class_name => 'Admin::Scholarship',inverse_of: :admin_scholarship_group
+  has_many :scholarships, :class_name => 'Admin::Scholarship',inverse_of: :group, foreign_key: 'group_id'
```

Comment:
Why do you do it ? Does the story tell you to it ? Please discard this changes. Because it has already an issue which already is being pending for your review.